code/modules/research/designs/weapon_designs.dm

```diff
@@ -3,37 +3,33 @@
 /////////////////////////////////////////
 
 /datum/design/nuclear_gun
-	name = "Advanced Energy Gun"
-	desc = "An energy gun with an experimental miniaturized reactor."
+	name = "Advanced Energy Gun Parts Kit"
+	desc = "A kit for an energy gun with an experimental miniaturized reactor."
 	id = "nuclear_gun"
 	req_tech = list("combat" = 5, "magnets" = 5, "powerstorage" = 5)
 	build_type = PROTOLATHE
 	materials = list(MAT_METAL = 10000, MAT_GLASS = 2000, MAT_URANIUM = 3000, MAT_TITANIUM = 1000)
-	build_path = /obj/item/gun/energy/gun/nuclear
-	locked = 1
+	build_path = /obj/item/weaponcrafting/gunkit/nuclear
 	category = list("Weapons")
 
 /datum/design/decloner
-	name = "Decloner"
+	name = "Decloner Parts Kit"
 	desc = "Your opponent will bubble into a messy pile of goop."
```

Comment:
I love this description but you say in all the others that it's "a kit for...", you might just want to standardize this

Reply:
This isn't the actual items parts kit description, just a short blob of text that you see in the protolathes UI before printing the kit. I think it's fine so long as it has parts kit in the actual name.